src/service/QuestionGenerator.js

```diff
@@ -0,0 +1,50 @@
+import { randomNumberInRange } from './randomNumberInRange';
+import {QuestionService} from './QuestionService';
+import { WHO_IS_THAT_POKEMON } from "./modes";
+
+const questionService = new QuestionService()
+
+export class QuestionGenerator {
+
+    constructor(mode) {
+        this.mode = mode;
+        this.minPokeId = 1;
+        this.maxPokeId = 152;
+        this.askedQuestions = [];
```

Comment:
`askedQuestions` is not yet used :)

Reply:
I fixed it. "askedQuestions" array was used. 

https://github.com/lukaszdutka/CodersCamp2020.Project.JavaScript.pokemonquiz/pull/22/commits/683d757dbc2ae9bb2ed292f1d569011544127823